src/main/java/net/earthcomputer/clientcommands/command/NetherCommand.java

```diff
@@ -0,0 +1,54 @@
+package net.earthcomputer.clientcommands.command;
+
+import com.mojang.brigadier.Command;
+import com.mojang.brigadier.CommandDispatcher;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.minecraft.client.resource.language.I18n;
+import net.minecraft.text.MutableText;
+import net.minecraft.text.Text;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+import static dev.xpple.clientarguments.arguments.CBlockPosArgumentType.blockPos;
+import static dev.xpple.clientarguments.arguments.CBlockPosArgumentType.getCBlockPos;
+import static net.earthcomputer.clientcommands.command.ClientCommandHelper.getLookCoordsTextComponent;
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.argument;
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.literal;
+
+public class NetherCommand {
+
+    public static void register(CommandDispatcher<FabricClientCommandSource> dispatcher) {
+        dispatcher.register(literal("cnether")
+            .then(argument("pos", blockPos())
+                    .executes(ctx -> convertCoords(ctx.getSource(), getCBlockPos(ctx, "pos"))))
+            .executes(ctx -> convertCoords(ctx.getSource(),
+                    ctx.getSource().getPlayer().getBlockPos())));
+    }
+
+    private static int convertCoords(FabricClientCommandSource source, BlockPos pos) {
+        BlockPos targetPos;
+        String sourceWorld;
+        String targetWorld;
+
+        if (source.getPlayer().world.getRegistryKey() == World.NETHER) {
+            targetPos = new BlockPos(pos.getX()*8, pos.getY(), pos.getZ()*8);
+            sourceWorld = I18n.translate("commands.cnether.world.nether");
+            targetWorld = I18n.translate("commands.cnether.world.overworld");
+        } else {
+            targetPos = new BlockPos(pos.getX()/8, pos.getY(), pos.getZ()/8);
+            sourceWorld = I18n.translate("commands.cnether.world.overworld");
+            targetWorld = I18n.translate("commands.cnether.world.nether");
+        }
+
+        source.sendFeedback(getCoordsTextComponent(pos)
+                .append(Text.translatable("commands.cnether.coords.left", sourceWorld, targetWorld))
+                .append(getLookCoordsTextComponent(targetPos))
+                .append(Text.translatable("commands.cnether.coords.right", sourceWorld, targetWorld)));
```

Comment:
Prefer `TranslatableText` arguments over appending text together.

Reply:
Well, this was based around the existing `getLookCoordsTextComponent()` will will return properly formatted coordinates, with a onClick event to send a `/clook` command. That means it *must* be a separate Text node, or the withClickEvent would apply to the entire string. So I needed at least three separate Text blocks -- the "look coords" and the block prior and after.

Due to the symmetry of the response, I thought it made more sense to keep this symmetry in how the feedback string was built up. We're not talking performance critical code here, so I opted to strive for good code readability instead.

Reply:
I don't see what you mean, as both solutions would allow separate click events for the coordinates. My suggestion was to use a single `commands.cpos.coords` rather than the separate `commands.cpos.coords.left` and `commands.cpos.coords.right`.

Reply:
That's a valid issue however I reckon we should solve it after the PR has been merged, because it's a problem around the code base.